FrenchFilter/sections/whitelist.txt

```diff
@@ -8,6 +8,8 @@ comandotorrents.com#@#a[href^="http://www.masterads.info/"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39013
 @@||dl.opensubtitles.org/*/download^
 @@||api.opensubtitles.org^$third-party
+! https://github.com/AdguardTeam/AdguardFilters/issues/41309
+@@||out.pladform.ru/player$domain=liveam.tv
```

Comment:
which filter blocks it?

Reply:
`AdGuard French`

Reply:
This rule here: ||out.pladform.ru^.
Rule comes from Liste FR.

Reply:
this is a bad rule. out.pladform.ru should not be blocked completely by general rule. Better exclude it.